frontend/templates/frontend/index.html

```diff
@@ -0,0 +1,19 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="/styles/index.css">
+  <title>Your farm | Are We Snek or Are We Dancer</title>
+</head>
+<body>
+
+  <section>
+    <div class="container">
+      <div id="app" class="columns"></div>
```

Comment:
Is `.columns` doing anything here? Do we have a `/styles/index.css` file?

Reply:
No, i think that was taken directly from the tutorial. I'll remove the class instead of implementing any styles because we'll likely remove the whole table of farms at some point anyway :)